### PR TITLE
install createrepo in docker image for RPM yum repo support

### PR DESCRIPTION
**Pull request by tianleh**
Signed-off-by: Tianle Huang <tianleh@amazon.com>

### Description

This is to support RPM yum repo.

https://github.com/opensearch-project/opensearch-build/issues/1549#event-6412585889
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docker/ci/dockerfiles/build.centos7.opensearch.x64.arm64.dockerfile [hunk 1]
```diff
@@ -114,7 +114,7 @@ CMD ["/bin/bash", "-l"]
 
 # Install ruby / rpm / fpm related dependencies
 RUN . /etc/profile.d/rvm.sh && rvm install 2.4.0 && rvm --default use 2.4.0 && \
-    yum install -y rpm-build && \
+    yum install -y rpm-build createrepo && \
```
Comment (peterzhuamazon) [inline docker/ci/dockerfiles/build.centos7.opensearch.x64.arm64.dockerfile:117]: Can you add `yum clean all` after createrepo?
Thanks.

Comment (tianleh) [reply docker/ci/dockerfiles/build.centos7.opensearch.x64.arm64.dockerfile]: updated

**Comment by peterzhuamazon** [comment]
Building image now will post the images here.

**Comment by peterzhuamazon** [comment]
@tianleh https://hub.docker.com/r/opensearchstaging/ci-runner/tags?page=1&ordering=last_updated&name=ci-runner-centos7-opensearch-build-v1

**Comment by tianleh** [comment]
> @tianleh https://hub.docker.com/r/opensearchstaging/ci-runner/tags?page=1&ordering=last_updated&name=ci-runner-centos7-opensearch-build-v1

spin up a new container and verify that `createrepo` is available. @peterzhuamazon 

```
[opensearch@fac4124e31e6 ~]$ cd opensearch-build/
[opensearch@fac4124e31e6 opensearch-build]$ ls
[opensearch@fac4124e31e6 opensearch-build]$ createrepo 
Error: Must specify a directory to index.
Usage: genpkgmetadata.py [options]

[opensearch@fac4124e31e6 opensearch-build]$ 
```

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**